docs/UserGuide.md

```diff
@@ -359,40 +395,140 @@ A person can have any number of tags and insurance policies (including 0).
 [Return to Table of Contents](#table-of-contents)
 <br><br>
 
+### <span style="color:#b573c9"><code>delete</code>: Delete client contact</span>
+
+**Purpose**: Deletes a specified client's contact from ClientBook.
+
+**Format**: `delete INDEX`
+
+* Deletes the client at the specified `INDEX`.
+* `INDEX` refers to the index number shown in the displayed client list.
+* `INDEX` must be 1 or higher, and less than or equal to the index of the last item in the displayed list.
+
+**Examples**:
+* `delete 2` deletes the 2nd person in the currently displayed list.
+
+* `find n/Charlotte` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+
+[Return to Table of Contents](#table-of-contents)
+<br><br>
+
+### <span style="color:#b573c9"><code>batch</code>: Execute commands in batch</span>
 
-### `list`: List all clients 
+**Purpose**: Executes the same `edit` or `delete` commands for multiple client contacts so that you do not have to
+enter the same command multiple times.
 
-**Purpose**: Shows a list of all clients in ClientBook. Optional identifiers can be specified to show a list with only the desired attributes.
+**Format**: `batch COMMAND INDICES [ARGUMENTS]`
+
+* Only `edit` and `delete` commands can be executed in batch.
+    * For more information on how these commands work and their parameters, refer to the
+      [`edit`](#edit-edit-client-contact) and [`delete`](#delete-delete-client-contact) sections accordingly.
+* `INDICES` are comma-separated e.g. `1,2,3` or `1, 2, 3` and they refer to the index number shown in the displayed client list.
+* `INDICES` must be 1 or higher, and less than or equal to the index of the last item in the displayed list.
+* The optional `ARGUMENTS` input is only applicable if the `COMMAND` is `edit`.
+* For `edit`, you can only batch edit the following attributes:
+    * phone number
+    * address
+    * tags
+    * insurance policies
+
+<div markdown="block" class="alert alert-info">
+:bulb: 
+**Tip**:
+The same rules apply when editing clients' phone number and address! <br>
+If a parameter is expected only once in the command, but you specified it multiple times, **only the last occurrence** of the parameter will be taken. 
+</div>
+
+**Examples**:
+* Updating the particulars of clients belonging to the same family and owning the same insurance policy.
+    * `batch edit 1, 2, 4 p/61234567 a/Hougang Green t/TanFamily i/FamPol#111`
+      <br><br>
+* To batch delete more than 1 client contact.
```

Comment:
To be consistent, if we are not putting spaces between examples, we should remove these here too.